capstone/src/main/webapp/about-us.html

```diff
@@ -44,6 +44,8 @@ <h3>Labels:</h3>
           <input type="submit">
         </form>
         <form hidden action="/club-edit" method="POST" id="edit-form" name="edit-about">
+          <h3 hidden id='requests-header'>Requests to join club</h3>
```

Comment:
No worries if this will take too long, but I was thinking - would it be better to have the "Requests to join" list separate from the edits? I think it'd be nicer to not have to click on edit to approve requests, but it's up to you

Reply:
I agree this would look a lot nicer, I thought some sort of pop up window would be really cool too, but I think I'll just keep it this way for the sake of time and maybe it could be something to add to the future improvements